recorded_future/plugin.spec.yaml

```diff
@@ -27,6 +27,54 @@ hub_tags:
 enable_cache: false
 language: python
 types:
+  list:
+    id:
+      title: ID
+      type: string
+      description: ID
+      required: false
+    name:
+      title: Name
+      type: string
+      description: Name
+      required: false
+    type:
+      title: Type
+      type: string
+      description: Type
+      required: false
+  enterpriseLists:
+    added:
+      title: Added
+      type: string
+      description: Added
+      required: false
+    list:
+      title: List
+      type: list
+      description: List
+      required: false
+  threatLists:
```

Comment:
```suggestion
  threatList:
```

Reply:
@mrinehart-r7 
There is the same reason as in EnterpriseLists.